scripts/jenkins/ardana/ansible/repositories.yml

```diff
@@ -54,3 +58,23 @@
   # FIXME: Resolve the file conflicts in the ardana packages and use the zypper module
   - name: Install ardana pattern
     command: zypper -n in patterns-cloud-ardana
+
+  # This can't happen earlier because we rely on the ardana pattern for creating
+  # the ardana user/group
+  - name: Ensure /etc/ardana exists
+    file:
+      path: /etc/ardana
+      state: directory
+      owner: ardana
+      group: ardana
+
+  - name: Capture build number of cloudsource media
+    get_url:
+      url: "{{ cloudsource_url }}/media.1/build"
+      dest: /etc/ardana/media-build-version
```

Comment:
/etc/openstack/ardana-media-buildid might be another variant, but we're well in the nit picking area now..